twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -0,0 +1,322 @@
+
+            instances = image.get('instances')
+            instance = instances[0]
+            image_name = instance.get('image')
+            if not image_name:
+                continue
+            if image_name.startswith(DOCKERIO_PREFIX):
+                image_name = image_name[len(DOCKERIO_PREFIX):]
+            image_tags = ["scanned_image:" + image_name] + self.config.tags
+
+            self._report_layer_count(image, namespace, image_tags)
+            self._report_service_check(image,
+                                       namespace,
+                                       SCAN_DATE_FORMAT,
+                                       tags=image_tags,
+                                       message="Last scan: " + image.get("scanTime"))
+            self._report_vuln_info(namespace, image, image_tags)
+            self._report_compliance_information(namespace, image, image_tags)
+
+    def report_hosts_scan(self):
+        namespace = self.NAMESPACE + ".hosts"
+        service_check_name = self.NAMESPACE + ".can_connect"
+        try:
+            scan_result = self._retrieve_json("/api/v1/hosts")
+            self.service_check(service_check_name, AgentCheck.OK, tags=self.config.tags)
+        except Exception as e:
+            self.warning("cannot retrieve registry data: {}".format(e))
+            self.service_check(service_check_name, AgentCheck.CRITICAL, tags=self.config.tags)
+            return None
+
+        for host in scan_result:
+            if 'hostname' not in host:
+                continue
+
+            hostname = host['hostname']
+            host_tags = ["scanned_host:" + hostname] + self.config.tags
+
+            self._report_service_check(host,
+                                       namespace,
+                                       SCAN_DATE_FORMAT,
+                                       tags=host_tags,
+                                       message="Last scan: " + host.get("scanTime"))
+            self._report_vuln_info(namespace, host, host_tags)
+            self._report_compliance_information(namespace, host, host_tags)
+
+    def report_container_compliance(self):
+        namespace = self.NAMESPACE + ".containers"
+        service_check_name = self.NAMESPACE + ".can_connect"
+        try:
+            scan_result = self._retrieve_json("/api/v1/containers")
+            self.service_check(service_check_name, AgentCheck.OK, tags=self.config.tags)
+        except Exception as e:
+            self.warning("cannot retrieve registry data: {}".format(e))
+            self.service_check(service_check_name, AgentCheck.CRITICAL, tags=self.config.tags)
+            return None
+
+        for container in scan_result:
+            if '_id' not in container:
+                continue
+
+            container_tags = []
+            container_info = container.get('info', {})
+            name = container_info.get('name')
+            if name:
+                container_tags += ["container_name:" + name]
```

Comment:
```suggestion
                container_tags.append("container_name:{}".format(name))
```

Reply:
I maintain that I like this better than the .append

Reply:
This is creating an intermediate list for no reason